spec/environment_spec.rb

```diff
@@ -545,6 +550,131 @@ def promote_content(content1, env1)
     }
   end
 
+  it 'should delete consumer together with his last environment' do
```

Comment:
This test already covers what `should cleanup consumers of deleted environment` is asserting, so I think we should remove that test in favor of this one

Reply:
Done.